crypto/lhash/lhash.c

```diff
@@ -72,11 +72,17 @@ void *OPENSSL_LH_insert(OPENSSL_LHASH *lh, void *data)
     unsigned long hash;
     OPENSSL_LH_NODE *nn, **rn;
     void *ret;
-
+    int val = 0;
+    
     lh->error = 0;
     if (lh->up_load <= (lh->num_items * LH_LOAD_MULT / lh->num_nodes))
-        expand(lh);
-
+        val = expand(lh);
+    
+    if (val != 0) {
```

Comment:
> In function 'expand', return 0 indicate success and return -1/1 indicate failure.

Afaics, it's 0 for success and -1 for failure. You don't use 1, and you don't need two different error returns. So, please use 1 for success and 0 for failure as is more customary in our project.